bridges/snowbridge/pallets/outbound-queue-v2/src/lib.rs

```diff
@@ -77,10 +77,14 @@ use bridge_hub_common::{AggregateMessageOrigin, CustomDigestItem};
 use codec::Decode;
 use frame_support::{
 	storage::StorageStreamIter,
-	traits::{tokens::Balance, EnqueueMessage, Get, ProcessMessageError},
+	traits::{
+		tokens::Balance,
+		EnqueueMessage, Get, ProcessMessageError,
+	},
 	weights::{Weight, WeightToFee},
 };
-use snowbridge_core::{ether_asset, BasicOperatingMode, PaymentProcedure, TokenId};
+use pallet_bridge_relayers::RewardLedger;
```

Comment:
Import from primitives crate:

```suggestion
use bp_relayers::RewardLedger;
```

Reply:
Addressed in https://github.com/vgeddes/polkadot-sdk/pull/12/commits/6ae7167648e00cc0f4f78262c3dc715315170b65.